extensions/3DTILES_implicit_tiling/README.md

```diff
@@ -0,0 +1,946 @@
+
+When all tiles exist, then their availability can be encoded by setting `tileAvailability.constant` to `1`, without needing an explicit bitstream.
+
+Only some tiles have content, and `contentAvailability.bufferView` indicates where the bitstream for the content availability is stored: The `bufferView` with index 0 refers to the `buffer` with index 0. This buffer does not have a `uri` property, and therefore refers to the _internal_ buffer. The `byteOffset` and `byteLength` indicate that the content availability bitstream is stored in the bytes `[0...11)` of the internal buffer.
+
+Some child subtrees exist, so `childSubtreeAvailability.bufferView` refers to another bitstream. The `bufferView` with index 1 refers to the buffer with index `1`. This buffer has a `uri` property, indicating that this second bitstream is stored in an external binary file.
+
+### Availability Packing
+
+Availability bitstreams are packed in binary using the format described in the [Booleans](../../specification/Metadata#booleans) section of the Cesium 3D Metadata Specification.
+
+## Glossary
+
+* **availability** - Data specifying which tiles/contents/child subtrees exist within a single subtree.
+* **bitstream** - A boolean array stored as a sequence of bits rather than bytes.
+* **bounding volume** - The spatial extent enclosing a tile or a tile's content, as defined in the [3D Tiles specification](../../specification#bounding-volumes).
+* **child subtree** - A subtree reachable from an available tile in the bottommost row of a subtree.
+* **content** - A content such as Batched 3D Model or Point Cloud as defined in the [3D Tiles specification](../../specification#introduction)
+* **implicit tiling** - A description of a tileset using recursive subdivision.
+* **implicit root tile** - A tile with the `3DTILES_implicit_tiling` extension, which denotes the root of an implicit tileset.
+* **octree** - A 3D subdivision scheme that divides each bounding volume into 8 smaller bounding volumes along the midpoint of the `x`, `y`, and `z` axes.
+* **quadtree** - A 2D subdivision scheme that divides each bounding volume into 4 smaller bounding volumes along the midpoint of the `x` and `y` axes.
+* **subtree** - A fixed-size section of the tree that contains availability information.
+* **subtree file** - A binary file storing information about a specific subtree.
+* **subdivision scheme** - A recursive pattern of dividing a parent tile into smaller child tiles occupying the same area. This is done by uniformly dividing the bounding volume of the parent tile.
+* **template URI** - A URI pattern containing tile coordinates for directly addressing tiles.
+* **tile** - A division of space that may contain content.
+* **tileset** - A hierarchical collection of tiles.
+* **tileset JSON** - A JSON file describing a tileset, as defined in the [3D Tiles specification](../../specification#tileset-json).
+
+## Extension JSON Schema Reference
```

Comment:
We decided to remove the JSON schema reference from other extensions, so it can be removed from here.